dashboard/config/puma.rb

```diff
@@ -26,7 +26,18 @@
   Cdo::AppServerHooks.before_fork
 end
 
+# Statsig is initialized here for managed environments. For development, it is
+# intialized in config/initializers/statsig.rb
+require 'statsig'
+# Determine if this is a managed test environment
+managed_test_environment = CDO.running_web_application? && CDO.test_system?
+# Enable local_mode for all environments except prod and test. This limits the
```

Comment:
Probably worth being very specific about what we mean when we say “test”. What we mean is that StatSig will be enabled on the managed test server’s web application server. We should be clear that unit tests, ActiveJobs, cron jobs, and the interactive ruby console (`dashboard-console`, `pegasus-console`) that execute on the managed test server will use StatSig in stubbed (local mode) and also that all code that executes in continuous integration builds (“Drone”) will use StatSig local mode.